src/coreclr/src/pal/src/libunwind/src/arm/Gresume.c

```diff
@@ -57,9 +57,12 @@ arm_local_resume (unw_addr_space_t as, unw_cursor_t *cursor, void *arg)
       };
 
       asm __volatile__ (
+        ".arm\n"
```

Comment:
I wonder if this is going to work at all. The whole function is compiled as thumb2 and the .arm pseudo instruction only influences the encoding of instructions. So we would get invalid instruction codes here.

Also, I would prefer change like this to be made in the libunwind repo first so that we don't diverge from it. I actually wonder how this works in the libunwind itself, as it is used on armv6 distros too.

Reply:
Looking at the changes, last two are bad. This change is in the working set.

Reply:
"The whole function is compiled as thumb2 and the .arm pseudo instruction only influences the encoding of instructions. "

The arm/thumb declaration at the top says globally all functions in this file are of this type. It doesn't prevent a specific function to specialize in my experience.